src/main/java/com/xpn/xwiki/plugin/lucene/IndexRebuilder.java

```diff
@@ -125,71 +137,71 @@
   private boolean onlyNew = false;
```

Comment:
must be volatile. Is accessed from multiple threads outside of synchronized blocks.